README.md

```diff
@@ -89,7 +95,7 @@ $ yarn run develop
 See it in a browser by opening up
 
 ```
-http://localhost:6789/
+http://localhost:9000/
```

Comment:
Actually, looks like I missed this as part of #19.  We can safely remove this instruction to manually open the browser as part of the _develop_ task.